home-application/src/main/java/com/softserveinc/ita/homeproject/application/config/HomeUserWrapperDetails.java

```diff
@@ -31,8 +30,8 @@ public HomeUserWrapperDetails(User user, Collection<Permission> userPermissions)
     public Collection<? extends GrantedAuthority> getAuthorities() {
         Set<GrantedAuthority> authorities;
         authorities = userPermissions.stream()
-            .map(permission -> new SimpleGrantedAuthority(permission.getName()))
-            .collect(Collectors.toSet());
+                .map(SimpleGrantedAuthority::new)
```

Comment:
Will this mapping assign a role value to the field "String role" in the new SimpleGrantedAuthority object?